### ISSUE TITLE
Widget to enable/disable tracking

### ISSUE THREAD

**Issue by israel-lugo** [redfish64/TinyTravelTracker#49]
Hi there,

First, let me congratulate you on a very nice application! I've just installed this yesterday on my One Plus 2 (OxygenOS 2.2.1, Android 5.1.1) and it's working perfectly. I've only encountered one problem so far (issue #47), but it's not a show stopper.

One suggestion: it would be nice to have a widget to quickly enable or disable the GPS tracking.

My use case would be to disable tracking after I reach home, since I'll be in the same place for 12 hours or so and it won't be easy to get GPS signal anyway. Also I'm usually at basement level at work, so for several hours there won't be any GPS signal either.

I realize this can be done via the settings, but it takes several clicks and I have to enter the password, so it's not as practical. If I could just toggle tracking with a single click, I could happily enable it while on the move, and disable it when I'm going to be stationary for a long time.

**Comment by collectorgeneral** [comment]
@redfish64 Agreed, this is a good idea. I have the same activity profile as him, so a toggle widget would be a good addition indeed.

**Comment by redfish64** [comment]
That is also what I was thinking.

I also am thinking about an algorithm change that reduces the gps usage
when the phones accelerometer and orientation sensors indicate the phone
hasn't moved.

Sorry, but I'm kind of busy with another project at the moment but maybe in
a couple of weeks I can get to this.

**Comment by israel-lugo** [comment]
I like @collectorgeneral's idea, a stop button in the notification would work. Regarding the display of additional information, that might be interesting as well. I would suggest, however, making sure that it doesn't create additional CPU requirements, such as having to wake up to update the information. I suppose that it could be updated while taking a GPS reading, since you're already awake for that and that's when you have new information anyway.

**Comment by collectorgeneral** [comment]
@redfish64 

Regarding the idea of reducing GPS usage when not moving, I read that you can query points from the system using the mindistance parameter. Have you considered that? [Link](https://developer.android.com/reference/android/location/LocationManager.html)
You could set the mindistance parameter to something reasonable (or allow the user to set it).

Although the documentation says that is is harder to conserve power using that parameter, I am curious to know if it is a better solution than the idea you had in mind,
